SUMMARY.md

```diff
@@ -1,6 +0,0 @@
-# Summary
```

Comment:
why'd we remove this and the book.toml?

Reply:
Wasn't sure they were being used 😅 
I'll add them back in.